dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/src/org/apache/cxf/jaxws/handler/soap/SOAPHandlerInterceptor.java

```diff
@@ -124,6 +139,9 @@ public void handleMessage(@Sensitive SoapMessage message) { // Liberty Change
             if (!chainAlreadyContainsSAAJ(message)) {
                 SAAJ_OUT.handleMessage(message);
             }
+            if (isFinestEnabled) {
+                LOG.finest(ending + " will be added to interceptor chain for outbound message.");   // Liberty Change issue #26529
```

Comment:
This may be duplicate trace, we should already be logging a message for each interceptor that gets added to the chain.